providers/src/airflow/providers/edge/CHANGELOG.rst

```diff
@@ -27,6 +27,10 @@
 Changelog
 ---------
 
+0.2.0pre0
+.........
+
```

Comment:
Can you add some details into changelog as well?

Reply:
Yes, I'll add some